Service/Model.Fake/FakeModelHelper.cs

```diff
@@ -0,0 +1,41 @@
+﻿using TodoSystem.Service.Model.Interface;
+
+namespace TodoSystem.Service.Model.Fake
+{
+    /// <summary>
+    /// Special help class for create new instance DTO.
+    /// </summary>
+    public static class FakeModelHelper
```

Comment:
Yep, copy constructor decision is better than helper that was temporary decision for imitation of serialization/deserialization data. But after wcf-host implemention there aren't reasons for local copy of DTO objects.